test/integration/create-next-app/templates/app.test.ts

```diff
@@ -222,4 +223,32 @@ describe.skip('create-next-app --app (App Router)', () => {
       })
     })
   })
+
+  it('should enable turbopack on --turbo flag', async () => {
```

Comment:
Th test passes, but the `describe` on top is skipped since #65845

Reply:
hello good work :heart: , I did a study, I wonder how your test coverage does not get stuck in timeout, I saw that you also got stuck in timeout 

[test/integration/create-next-app/templates/app.test.ts](https://github.com/vercel/next.js/pull/65927/files#diff-3a871014cf9ea1a26d1002d0ea9c5bf5226cbbfb7ebd5cda02b922b1460e0ff2R227)